android-components/components/ui/icons/src/main/res/drawable/mozac_ic_information_fill_24.xml

```diff
@@ -1,13 +1,16 @@
-<?xml version="1.0" encoding="utf-8"?>
 <!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
 <vector xmlns:android="http://schemas.android.com/apk/res/android"
-    android:width="14dp"
-    android:height="14dp"
-    android:viewportHeight="24.0"
-    android:viewportWidth="24.0">
+    xmlns:tools="http://schemas.android.com/tools"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="24"
+    android:viewportHeight="24">
     <path
+        android:pathData="M12,22c5.523,0 10,-4.477 10,-10S17.523,2 12,2 2,6.477 2,12s4.477,10 10,10zM12.875,7v1.75h-1.75L11.125,7h1.75zM12.875,17v-6.25h-1.75L11.125,17h1.75z"
```

Comment:
v1.5.0 just got released 43 minutes ago so we'll want to re-sync the pathData with the latest at https://github.com/FirefoxUX/acorn-icons/blob/main/icons/mobile/24/xml/ic_information_fill_24.xml